system/cfme-setup.sh

```diff
@@ -19,11 +19,17 @@ sed -i 's@ACTIVE_CONSOLES=/dev/tty\[1-6\]@ACTIVE_CONSOLES=/dev/tty3@' /etc/sysco
 # httpd needs to connect to backend workers on :3000 and :4000
 setsebool -P httpd_can_network_connect on
 
+# Default value for MIQ_APACHE_ROOT_DIR is / unless overridden
+# via an enviornmental variable in /etc/default/evm for SCL Apache
+
+: ${MIQ_APACHE_ROOT_DIR:="/"}
```

Comment:
Again, I would prefer to point to the config directory since we don't appear to be using anything but the config directory.